easybuild/easyconfigs/i/Instant/Instant-1.5.0-goolf-1.5.14-Python-2.7.8.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = "PythonPackage"
+
+name = 'Instant'
+version = '1.5.0'
+
+homepage = 'https://launchpad.net/instant'
+description = """Instant is a Python module that allows for instant inlining of 
+ C and C++ code in Python. It is a small Python module built on top of SWIG and Distutils.
+ It is part of the FEniCS Project (http://fenicsproject.org)."""
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+
+#unfort. this doesn't work, as the file has a very long and strange file name when downloaded...
```

Comment:
OK, we have the same problem with downloads from google drive. You add a comment where you have to download it